src/components/OnlinePeersView/OnlinePeer/OnlinePeer.js

```diff
@@ -24,7 +24,7 @@ const OnlinePeer = ({
             subheader: (
               <div>
                 <p>Day: {integerString(dayCount)}</p>
-                <p>Level: {integerString(levelAchieved({ experience }))}</p>
+                <p>Level: {integerString(levelAchieved(experience))}</p>
```

Comment:
We'll need to ensure that `experience` has a fallback value because this data comes from online peers. When this PR is released, it will take some time for preexisting installs to update to use the new experience system.

```suggestion
                {/* TODO: Remove `?? 0` after 10/24 */}
                <p>Level: {integerString(levelAchieved(experience ?? 0))}</p>
```